exercises/tournament/description.md

```diff
@@ -60,8 +60,3 @@ Devastating Donkeys;Courageous Californians;draw
 ```
 
 Means that the Devastating Donkeys and Courageous Californians tied.
-
-Your program should only process input lines that follow this format.
```

Comment:
I'm with @Insti: omitting that line about validity seems to make the most sense to me.